### PR TITLE
Add Job Name field to AJAX Spider Automation dialogue

### PR DESCRIPTION
**Pull request by amankumarchagti**

### CODE REVIEW

File: addOns/spiderAjax/src/main/resources/org/zaproxy/zap/extension/spiderAjax/resources/Messages.properties [hunk 1]
```diff
@@ -166,3 +166,4 @@ spiderajax.toolbar.button.start     = New Scan
 spiderajax.toolbar.button.stop      = Stop AJAX Spider
 spiderajax.outofscope.response      = (403 Forbidden) Out of AJAX Spider scope
 spiderajax.warn.message.phantomjs.bug.invalid.target = The spider will not run.\nA bug in PhantomJS prevents the requests to localhost,\n127.0.0.1 and ::1 from being proxied through ZAP.
+spiderajax.automation.dialog.ajaxspider.name= Job Name
```
Comment (thc202) [inline addOns/spiderAjax/src/main/resources/org/zaproxy/zap/extension/spiderAjax/resources/Messages.properties:169]: This should be added closer to existing messages.

**Comment by thc202** [comment]
The changelog should be updated.
The code should be properly formatted (use the Gradle task `spotlessApply`).

File: addOns/spiderAjax/src/main/java/org/zaproxy/zap/extension/spiderAjax/AjaxSpiderDialog.java [hunk 2]
```diff
@@ -65,7 +65,8 @@
     private static final String FIELD_DURATION = "spiderajax.options.label.maxduration";
     private static final String FIELD_EVENT_WAIT = "spiderajax.options.label.eventwait";
     private static final String FIELD_RELOAD_WAIT = "spiderajax.options.label.reloadwait";
-
+    
+
```
Comment (kingthorin) [inline addOns/spiderAjax/src/main/java/org/zaproxy/zap/extension/spiderAjax/AjaxSpiderDialog.java:69]: This white space change should be reverted

**Comment by amankumarchagti** [comment]
Okay..let me see

**Pull request closed without merging**